### ISSUE TITLE
Language levels in one's profile are wrongly rendered

### ISSUE THREAD

**Issue by Guybrush88** [Tatoeba/tatoeba2#1985]
Currently, the levels for each language within a user's profile are showing an issue in their appearance, since now stars have been replaced by some text. I'm attaching a screenshot taken from my own profile.
![Istantanea_2019-10-12_18-53-54](https://user-images.githubusercontent.com/1702194/66705006-f6184480-ed21-11e9-9a96-79ec5beec182.png)

**Comment by trang** [comment]
Did you install any particular addon on your Firefox?

**Comment by Guybrush88** [comment]
nope, no add-ons installed. Today I made some general changes to the browser's fonts, that's the only change I made locally. I hope this helps

**Comment by trang** [comment]
Your changes are probably what causes this issue. The icons that are no longer displayed properly are actually characters from a font.

**Comment by Guybrush88** [comment]
my suggestion, then, it's to make such icons independent to any font, since people might have different reasons to change it (i.e.: people with reading issues might want to use a more suitable font to improve the reading fluency, or maybe someone might want to change the global font because the new one appears better on his/her device)

**Comment by Yorwba** [comment]
Maybe the fallback text could use ★ black star (U+2605), ☆ white star (U+2606), 🖍 lower left crayon (U+1F58D) and 🗑 wastebasket (U+1F5D1).

**Comment by Guybrush88** [comment]
to sum up, all the other affected icons (at least the one I found) are in such pages:
- https://tatoeba.org/ita/sentences/of_user/Guybrush88
- https://tatoeba.org/ita/contributions/latest

More specifically:
- the arrows to switch between different pages
- the icons next to any possible action one can do within a profile from the user's sentences page
- the 'info' icon to open sentences in the 'latest contributions' page

For now, I didn't find anything else that is affected

**Comment by trang** [comment]
Note that all icons will be affected because we are slowly migrating towards using [Material icons](https://material.io/resources/icons/?style=baseline) everywhere we can.

These Material icons are all characters from a font. This is because we are using the [md-icon](https://material.angularjs.org/latest/api/directive/mdIcon) directive from AngularJS Material and it has been set up to read the icons from a font. 

It has been set up that way only for convenience on development side. In the long term, I agree that reading from SVGs would be more suitable.
